lib/shopify_transporter/exporters/magento/database_cache.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+require 'csv'
+require 'sequel'
+
+module ShopifyTransporter
+  module Exporters
+    module Magento
+      class DatabaseCache
```

Comment:
TBH, I don't think this is the right time for this abstraction. Especially considering it's use. I don't think it's blocking but I just wanted to point that out. I could see this being just a part of the options class. It's basically only used within there making it pretty much isolated in its use.

Reply:
I vaguely thought this too. I just didn't feel it strongly enough to block this PR over it.

Reply:
^ non-blocking

Reply:
👍 Thanks for the feedback folks! 

This came before `ProductOptions` and I didn't think about it.

Personally I was thinking we should wrap `DatabaseCache` and `DatabaseExporter` into a `VirtualDatabase`/some middle group, and use that instead of 2 classes for the database. I didn't get around to it :(

Reply:
Like I said above -- I'm ok with merging things the way they evolved, and changing them in the future as needs come up or as we have better ideas. I wouldn't push hard for you to remove this abstraction.